### DISCUSSION TITLE
Faster Terminal Emulator

### DISCUSSION THREAD

**Discussion by failafail8** [ddterm/gnome-shell-extension-ddterm#559] [General]
I know nothin about coding, and i have no need for a faster terminal. 
But i guess this article and its commentaries might be interesting for you.
https://www.phoronix.com/news/Linux-Terminals-Could-Be-Faster 
i learned about this Wayland native and fast ternimal named foot ( the name made my day :) )

**Comment by amezin** [comment]
Yeah, I've read that

Unfortunately, extensions on extensions.gnome.org can't include binaries, large 3rd party libraries, etc. So I'm essentially limited to the libraries shipped with GNOME Shell - and the only available option for terminals is VTE
